### PR TITLE
add 3 methods

### PR DESCRIPTION
**Pull request by papaloco1234**
1. get pair of stations from GTFS transfers,
   needs to run GTFSMain to generate the json file before calling ModifiedRaptor
2. getNumberofTrips in ModifiedRaptor
3. getTravelTime in ModifiedRaptor

### CODE REVIEW

**Comment by papaloco1234** [comment]
Hi Yi
in case you hasn't notice, This is Chiwen.
Please let me know are these methods works

**Comment by meiyi1986** [comment]
Hi Chiwen,

Thanks. I saw the changes. There is a firework problem at RMIT and I
couldn't connect the GitHub server. However, I believe it should be
working, since the revision is minor. Are you working on the next step? I
mean calculating the travel time and number of transfers for each region
using the modified RAPTOR. Do you have any problem with the region data
that were sent to you last time?

Cheers,
Yi

**Comment by papaloco1234** [comment]
Hi Yi,

yes, facing some problems.
while I was thinking and working on parsing the csv to json, saw something
odd.
It seems to me the full path may across more than one regions. correct?
If you can show me an example of a pair of stations and the expect result
will be much appreciated.
Or maybe I just get the whole idea wrong...

Cheers,
Chiwen

**Comment by meiyi1986** [comment]
Hi Chiwen,

In my opinion, the problem should be like this: let the city centre be
denoted as C, there are regions R1, R2, ..., around it. Let's suppose there
is only one stop in each region. Therefore, there should be a path from C
to each region, regardless whether it traverses other regions. Don't worry
about that. For example, when calculating the path from C to R2, one may
obtain the path C-R1-R2, which across R1. It doesn't matter as long as the
path to R2 is obtained along with the travel time and number of transfers.
So basically, the steps should be:
1. Calculate the path C-R_i for each R_i and record the travel time and
   number of transfers;
2. Sort the regions based on the travel time and transfers;
3. Select the best regions.

I'm not doing the region ranking thing, so don't know about the csv...

Cheers,
